core/src/main/se/tevej/game/input/TKeyBoard.java

```diff
@@ -0,0 +1,67 @@
+package main.se.tevej.game.input;
+
+import static com.badlogic.gdx.Input.Keys.*;
+
+public interface TKeyBoard {
+
+    TKeyBoard addClickedListener(OnClickedListener onClickedListener);
+
+    interface OnClickedListener {
```

Comment:
Why does an interface have an inner interface? If there does not exist an extremely good reason, please change.